### PR TITLE
AbixCastrO

### PR DESCRIPTION
**Pull request by a-ridley**
What do you think?

### CODE REVIEW

**Comment by pwoosam** [comment]
This is great! But could you move your servoTest1.ino file into a folder named AbixCastrO.

So your fork's root directory should look like this:
```
servo_example/
  servo_example.ino
AbixCastrO/
  servoTest1.ino
README.md
```

File: servoTest1.ino [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+void setup() {
+  servoVarName.attach(servoPin);
+}
+
+void loop() {
+  for (int pos =0; pos < 180; pos++) {
+    servoVarName.write(pos);
+    delay(20);
+  }
+  for (int pos = 180; pos > 0; pos-= 180/8) {
```
Comment (pwoosam) [inline servoTest1.ino]: I know this is a little picky, but could you fix the whitespace here?
`pos-= 180/8` should be `pos -= 180 / 8`

Comment (a-ridley) [reply servoTest1.ino]: Okay, thanks!

**Comment by pwoosam** [comment]
It looks like you've created a branch here. I'd like for you to fork this repository to your account (but it looks like you've already done that), then make changes to your master or create a branch on your fork and make changes to that.

Then you can submit a pull request using the master branch of your forked repository or another branch if you made one.

**Pull request closed without merging**